src/loaders/tiff/lib/indexers.ts

```diff
@@ -147,3 +149,18 @@ function getOmeIFDIndexer(
     }
   }
 }
+
+export function getMultiTiffIndexer(tiffs: MultiTiffImage[]) {
```

Comment:
Perhaps an idea for a follow-up PR, we could have the indexer dynamically create the GeoTIFFImage objects on first access. Right now we initialize all the images before this point.

Reply:
Probably an edge case and something you wouldn't notice unless there were many many separate urls/files.

Reply:
I'm having it load all of the GeoTIFFImages at first so I can check to make sure all of the images have the same width and height. I'd be happy to change if there's a way to do it without getting the images, or if we don't need that check.

Reply:
> I'm having it load all of the GeoTIFFImages at first so I can check to make sure all of the images have the same width and height. I'd be happy to change if there's a way to do it without getting the images, or if we don't need that check.

I think that's totally fine for now! Just thinking if we run into latency issues, this could be a place where we could be "smarter" (e.g., cache the first seen `width`/`height` and assert that the other loaded images are identical when first indexed).